CSS folder/index.css

```diff
@@ -0,0 +1,269 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Times New Roman", Times, serif;
+  line-height: 1.4;
+}
+
+a {
+  text-decoration: none;
+}
+
+p {
+  margin: 0.75rem 0;
+}
```

Comment:
Please include a line of space between CSS rules/blocks for better readability. You can use Prettier to automate this. I'll share an article on the group channel on how to go about this.

Reply:
Resolved.